pkg/minishift/network/settings.go

```diff
@@ -60,14 +175,124 @@ func printNetworkSettings(networkSettings NetworkSettings) {
 func fillNetworkSettingsScript(networkSettings NetworkSettings) string {
 	result := &bytes.Buffer{}
 
-	tmpl, err := template.New("networkScript").Parse(configureNetworkScriptTemplate)
-	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error creating network script template: %s", err.Error()))
+	tmpl := template.New("networkScript")
+
+	if networkSettings.Disabled {
+		tmpl.Parse(configureNetworkScriptDisabledAddressTemplate)
+	} else if networkSettings.UseDHCP {
+		tmpl.Parse(configureNetworkScriptDynamicAddressTemplate)
+	} else {
+		tmpl.Parse(configureNetworkScriptStaticAddressTemplate)
 	}
-	err = tmpl.Execute(result, networkSettings)
+
+	err := tmpl.Execute(result, networkSettings)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error executing network script template:: %s", err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error executing network script template: %s", err.Error()))
 	}
 
 	return result.String()
 }
+
+func determineDefaultGateway(ipaddress string) string {
+	ip := net.ParseIP(ipaddress)
+	ip = ip.To4()
+	//ip = ip.Mask(ip.DefaultMask())
```

Comment:
Let's remove commented code.

Reply:
sure, this I can agree to... but the previous cases are cases I left in with a reason. They are supposed to be addressed, but since our configuration code is still 'unclear' of moving forward, ...

Reply:
@gbraad   `//ip = ip.Mask(ip.DefaultMask())` is that intended?